app/styles/components/_imaging.scss

```diff
@@ -0,0 +1,2 @@
+.imaging-row .notes { max-width: 300px; }
+.completed-imaging-row .notes { max-width: 300px; }
```

Comment:
Yeah, if the value is the same (which it appears to be) and there is no other conflict, just use the existing classname.